src/lang/pt_BR.json

```diff
@@ -0,0 +1,751 @@
+  "Errors.map.placeNotFound": "Nenhum resultado encontrado por Nominatim.",
+  "Errors.widgetWorkspace.renderFailure": "Não é possível renderizar o espaço de trabalho. Mudando para um layout de trabalho.",
+  "Errors.josm.noResponse": "O controle remoto do OSM não respondeu. Você tem o JOSM em execução com o Controle Remoto ativado?",
+  "Errors.josm.missingFeatureIds": "Os recursos desta tarefa não incluem os identificadores OSM necessários para abri-los de forma autônoma no JOSM. Por favor, escolha outra opção de edição.",
+  "KeyMapping.openEditor.editId": "Editar no ID",
+  "KeyMapping.openEditor.editJosm": "Editar no JOSM",
+  "KeyMapping.openEditor.editJosmLayer": "Editar na nova camada do JOSM",
+  "KeyMapping.openEditor.editJosmFeatures": "Editar apenas recursos no JOSM",
+  "KeyMapping.openEditor.editLevel0": "Editar no nível 0",
+  "KeyMapping.layers.layerOSMData": "Alternar camada de dados do OSM",
+  "KeyMapping.layers.layerTaskFeatures": "Alternar Camada de Recursos",
+  "KeyMapping.layers.layerMapillary": "Alternar Camada Mapilar",
+  "KeyMapping.taskEditing.cancel": "Cancelar edição",
+  "KeyMapping.taskEditing.fitBounds": "Ajustar Mapa aos Recursos da Tarefa",
+  "KeyMapping.taskEditing.escapeLabel": "ESC",
+  "KeyMapping.taskCompletion.skip": "Pular",
+  "KeyMapping.taskCompletion.falsePositive": "Não é um problema",
+  "KeyMapping.taskCompletion.fixed": "Eu consertei isso!",
+  "KeyMapping.taskCompletion.tooHard": "Muito difícil / não consegui ver",
+  "KeyMapping.taskCompletion.alreadyFixed": "Já fixa",
+  "KeyMapping.taskInspect.nextTask": "Próxima tarefa",
+  "KeyMapping.taskInspect.prevTask": "Tarefa Anterior",
+  "Project.GroupType.admin": "Admin",
+  "Project.GroupType.write": "Escrever",
+  "Project.GroupType.read": "Ler",
+  "Challenge.sort.name": "Name",
+  "Challenge.sort.created": "Newest",
+  "Challenge.sort.popularity": "Popular",
+  "Challenge.sort.default": "Padrão",
+  "Task.loadByMethod.random": "Aleatória",
+  "Task.loadByMethod.proximity": "Proximidade",
+  "Task.priority.high": "Alto",
+  "Task.priority.medium": "Médio",
+  "Task.priority.low": "Baixo",
+  "Task.status.created": "Criado",
+  "Task.status.fixed": "Fixo",
+  "Task.status.falsePositive": "Não é um problema",
+  "Task.status.skipped": "Pulou",
+  "Task.status.deleted": "Excluído",
+  "Task.status.alreadyFixed": "Já fixa",
+  "Task.status.tooHard": "Demasiado difícil",
+  "Locale.en-US.label": "en-US (Inglês dos EUA)",
+  "Locale.es.label": "es (Español)",
+  "Locale.de.label": "de (Deutsch)",
+  "Locale.fr.label": "fr (Français)",
+  "Locale.af.label": "af (Afrikaans)",
+  "Locale.ja.label": "ja (日本語)",
+  "Locale.ko.label": "ko (한국어)",
+  "Dashboard.ChallengeFilter.visible.label": "Visível",
+  "Dashboard.ChallengeFilter.pinned.label": "Preso",
+  "Dashboard.ProjectFilter.visible.label": "Visível",
+  "Dashboard.ProjectFilter.owner.label": "Possuído",
```

Comment:
owner = Dono ou Proprietário

Reply:
Corrigi conforme sugestão! Obrigado! / Correct as per suggestion! Thank you!